# gains for yaw = 0
# position_gain: {x: 6, y: 6, z: 6}
# velocity_gain: {x: 3.8, y: 3.8, z: 3.8}
# attitude_gain: {x: 4, y: 4, z: 0}
# vehicle_mass: 0.606

# gains for desired yaw
position_gain: {x: 6, y: 6, z: 6}
velocity_gain: {x: 3.8, y: 3.8, z: 3.8}
attitude_gain: {x: 4, y: 4, z: 1.2}
vehicle_mass: 0.606